### DISCUSSION TITLE
2nd academic conference books

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#44] [General]
지난 논의 때 언급한 책을 보니 다음 학술 회의 때 얘기할 책이 정해졌더라고요

- [함께 자라기](http://aladin.kr/p/wLch8)
- [프로그래머의 길 멘토에게 묻다](http://aladin.kr/p/aFfS7)
- [실용주의 프로그래머](http://aladin.kr/p/RGT3Z)

TMI를 해 보자면

함께 자라기의 경우
윤우님과 얘기하다 보니 윤우님이 이미 구매해서 보고 있다는 얘기를 들었던 것 같습니다.
출판 하자 마자 구매했고, 회사에 비치되어 있는 책이기도 합니다.

프로그래머의 길 멘토에게 묻다의 경우는
제가 워낙에 멘티 분들에게 선물로 주는 책 1호로 점찍어 놓은 책이다 보니
윤우님, 혜인님에게는 이미 2년 전에 선물로 드렸었고,
제 회사에도 있는 책인데다 동훈님에게도 권해 드렸던 책입니다.

마지막으로 실용주의 프로그래머의 경우는
여러가지 case에 대해서 재미있는 얘기 그리고 패턴 들에 대해 얘기해 주고 있고
고전 축에 속하지만 꽤나 볼만한 책이라고 봅니다.
유일하게 저는 한번 읽은 책이고 회사에는 제가 주문한 책은 아니지만 이미 다른 분이 주문해 놓은 초판본이 있네요.

---

이 세 권이 다음 academic conference 책으로 선정되면 어떨까 해서 의견 남겨봅니다 ㅎㅎ
(물론 당연하게도 위 세 권의 책은 제가 이미 읽은 책입니다!)

**Comment by yoonucho** [comment]
좋아요! 유감없습니다!

**Comment by hi-hyein** [comment]
저도 아주 좋습니다~!
